pkg/engine/vulnerability_builder.go

```diff
@@ -243,12 +230,73 @@ func detectLine(file *model.FileMetadata, searchKey string, logWithFields *zerol
 	}
 
 	if foundAtLeastOne {
-		return currentLine + 1
+		return vulnerabilityLines{
+			line:     currentLine + 1,
+			vulnLine: getAdjacents(currentLine, outputLines, lines),
+		}
 	}
 
 	logWithFields.Warn().Msgf("failed to detect line, query response %s", searchKey)
 
-	return UndetectedVulnerabilityLine
+	return vulnerabilityLines{
+		line:     UndetectedVulnerabilityLine,
+		vulnLine: model.VulnLines{},
+	}
+}
+
+// getAdjacents is used to get the lines adjecent to the line that contains the vulnerability
+// adj is the amount of lines wanted
+func getAdjacents(idx, adj int, lines []string) model.VulnLines {
```

Comment:
would rename to getAdjacentLines to be more clear

Reply:
resolved